src/data/camera.ts

```diff
@@ -41,16 +41,18 @@ export const fetchThumbnailUrlWithCache = (
   entityId: string,
   width: number,
   height: number
-) =>
-  timeCachePromiseFunc(
+) => {
+  const itemKey = `${entityId}_${width}_${height}`;
```

Comment:
After thinking about this a bit more, I think it makes sense to add the width and height AFTER we get the url back from the cache since the url for the entity itself isn't going to change and we want to cache that before we append the width and height params